src/libcprover-rust/include/c_errors.h

```diff
@@ -2,6 +2,8 @@
 
 #pragma once
 
+#include <string>
```

Comment:
Why has this file being touched by this PR?
It shouldn't be changed.

Reply:
removed, sorry for cluttering